packages/cli/src/fulltext/FileIndex.ts

```diff
@@ -39,7 +39,7 @@ export class FileIndex {
     const searchExpr = queryKeywords(keywords).join(' OR ');
     const rows = this.database.prepare(query).all(searchExpr, limit);
     rows.forEach((row: any) => {
-      if (verbose()) console.log(`Found row ${row.file_name}`);
+      if (verbose()) console.warn(`Found row ${row.file_name}`);
```

Comment:
I love getting the answer on stdout too, but I recommend that for your scripting purposes you add a file output option. Because in the future someone is probably going to accidentally write to the `log` again.